docs/Troubleshooting-(v0.25.0+).md

```diff
@@ -201,6 +202,95 @@ communications. It may be necessary to examine firewall logs (perhaps with
+that a conservative antivirus engine may perceive the observed network
+behavior as consistent with that of ["backdoor"](https://en.wikipedia.org/wiki/Backdoor_(computing))
+malware that may reach out to arbitrary network destinations in the name of
+transmitting sensitive information or providing network channels for
+nefarious remote logins. Of course, the [functionality of Suricata-Update](https://suricata.readthedocs.io/en/suricata-6.0.0/rule-management/suricata-update.html)
+as described by the OISF is of course quite benign: In its default
+configuration as it's used by Brimcap/Brim, the network destinations it
+reaches out to are servers associated with providing the
+[Emerging Threats Open](https://rules.emergingthreats.net/open/) ruleset.
+
+Having established its benign nature, a likely follow-on concern would be
+whether it has been altered in the process of bundling it for use with the app.
+Indeed, the Suricata-Update tool in its original form is written in Python and
+in a normal Suricata install, users are responsible for having a compatible
+Python installation that's capable of running the tool. This seems a reasonable
+expectation for the types of servers on which Suricata is often installed.
+However, since it is our intent for Brim to be desktop software that's easy
+to install, we felt that bundling Python with Brim would bring excessive
+bloat, and we felt requiring users to maintain a working Python installation
+as a prerequisite for having valid alerts would be a heavy burden. It's for
+this reason that we made the decision to use the popular [PyInstaller](https://www.pyinstaller.org/)
+tool to "freeze" Suricata-Update into standalone executables that could be
+bundled with each platform and hence run without external dependencies.
+
+If you wish to perform your own audit to confirm that the executable has not
+been tampered with in the build process, below are some pointers to trace
+its origin.
+
+* A Brim-managed [fork of suricata-update](https://github.com/brimdata/suricata-update) holds a
+  copy of the [original OISF repo](https://github.com/OISF/suricata-update).
+  As of Brim
+  release `v0.25.0`, the fork is holding a copy of the OISF repo as of tag
+  `1.2.0`.
+
+* A branch [`fix-windows`](https://github.com/brimdata/suricata-update/tree/fix-windows)
+  exists in the Brim-managed repo that includes minimal changes that we found
+  necessary for the executable to build and run on Windows. The total set of
+  changes can be reviewed [here](https://github.com/brimdata/suricata-update/compare/1.2.0...0b10a83).
+
+* A [build-suricata repo](https://github.com/brimdata/build-suricata) contains automation that's
+  used to create the Suricata packages that are bundled with Brimcap. This
+  includes [script code](https://github.com/brimdata/build-suricata/blob/9847f9ba72b24a132aa7068a15b9d3ff9e63af1b/.github/workflows/windows-build.yaml#L103-L109)
+  that unpacks the enhanced Python and runs PyInstaller to generate the
```

Comment:
This reads as though we've modified Python rather than Suracata-Update.
```suggestion
  that unpacks the enhanced Suricata-Update and runs PyInstaller to generate the
```